### PR TITLE
missing tests for qr and lq

### PR DESCRIPTION
**Pull request by kshyatt**
Just trying to grab a few more missed methods...

### CODE REVIEW

File: stdlib/LinearAlgebra/test/qr.jl [hunk 1]
```diff
@@ -71,6 +71,11 @@ rectangularQ(Q::LinearAlgebra.AbstractQ) = convert(Array, Q)
                 rstring  = sprint((t, s) -> show(t, "text/plain", s), r)
                 qstring  = sprint((t, s) -> show(t, "text/plain", s), q)
                 @test qrstring == "$(summary(qra))\nQ factor:\n$qstring\nR factor:\n$rstring"
+                # iterate
+                q, r = qra
+                @test q*r ≈ a
+                # property names
+                @test Base.propertynames(qra)       == (:R, :Q)
```
Comment (dkarrasch) [inline stdlib/LinearAlgebra/test/qr.jl]: Not that it matters, but what is this extensive space good for? 😛

**Review by dkarrasch — APPROVED** [review]
LGTM. I'd say merge after tests pass.

File: stdlib/LinearAlgebra/test/qr.jl (hunk 1 above)
Comment (kshyatt) [reply stdlib/LinearAlgebra/test/qr.jl]: I must have hit tab...

**Comment by kshyatt** [comment]
OS X error was in `Serialization`, Windows error was a timeout

**Pull request merged**